### ISSUE TITLE
Centralized whitelist

### ISSUE THREAD

**Issue by LPGhatguy** [Roblox/cla-signature-bot#4]
It requires some duplicated effort to maintain an employee whitelist across many repositories in the same organization.

It would be great to support pulling the contributor whitelist from a central repository, perhaps the same repository that contains who has signed the CLA.

**Comment by cliffchapmanrbx** [comment]
Yep this would be a great feature to add. The primary way we were talking about implementing this was as a check against membership in the organization that owns the repository. I'm not sure if the provided GITHUB_TOKEN has the scope necessary to hit the organization API to request membership details so it might depend on #3 and properly implementing a GitHub App for auth instead of using a PAT.

**Comment by cliffchapmanrbx** [comment]
It is super unclear if this can be done using the GITHUB_TOKEN that comes out of the box. According to the [permission list](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token) the [metadata endpoint](https://developer.github.com/v3/apps/permissions/#metadata-permissions) is available which can read the list of members in the org. _However_ the info for the list members endpoint indicates [the requestor must be a member of the org](https://developer.github.com/v3/orgs/members/#list-organization-members) to get non-public information. So if the requestor is the github action does that count?

No idea. Gotta test.
